src/platform_impl/macos/app_state.rs

```diff
@@ -291,6 +276,15 @@ impl Handler {
         *self.control_flow.lock().unwrap()
     }
 
+    fn set_control_flow(&self, new_control_flow: ControlFlow) {
+        let mut control_flow = self.control_flow.lock().unwrap();
+
+        if let ControlFlow::Exit = *control_flow {
+        } else {
+            *control_flow = new_control_flow
+        }
```

Comment:
Could you rewrite such things to simply use `!matches!` or `!=`?

Reply:
Thanks, done!